src/ucp/core/ucp_context.h

```diff
@@ -237,6 +237,12 @@ typedef struct ucp_tl_iface_atomic_flags {
     } atomic32, atomic64;
 } ucp_tl_iface_atomic_flags_t;
 
+/* datatype used to store address bandwidth value.
+   optimization hint: if bandwidth > 0 - it is dedicated value,
+   else - shared value. Assuming that both values can't be not-null */
```

Comment:
`can't be 0` ?

Reply:
can be, but there is no matter which one is [non]zero

Reply:
this trick to encode bandwidth should be only inside address.c when packing/unpacking iface attr
in all other places it should be 2 variables